api/base/exceptions.py

```diff
@@ -83,6 +83,10 @@ def json_api_exception_handler(exc, context):
     return response
 
 
+class NotImplementedError(APIException):
+    status_code = status.HTTP_501_NOT_IMPLEMENTED
+    default_detail = _('This view is not yet implemented.')
```

Comment:
The term "view" is Python-specific and shouldn't be returned to the client. How about "This endpoint is not yet implemented."